stdlib/websub/src/main/ballerina/src/websub/commons.bal

```diff
@@ -304,59 +302,58 @@ public type Notification object {
 
     # Retrieves the type of the payload of the content delivery request (i.e: the `content-type` header value).
     #
-    # + return - Returns the `content-type` header value as a string
+    # + return - The `content-type` header value as a `string`
     public function getContentType() returns @tainted string {
         return self.request.getContentType();
     }
 
     # Extracts `json` payload from the content delivery request.
     #
-    # + return - The `json` payload or `error` in case of errors. If the content type is not JSON, an `error` is returned.
+    # + return - The `json` payload or else an `error` if the content type is not JSON
```

Comment:
Error can be returned as a result of getXXXPayload issue. You have removed that part and only talks about an `error` if the content type is not JSON.
Please fix it, others as well
Do not change the meaning of the docs